namespaces/live-1.cloud-platform.service.justice.gov.uk/case-notes-to-probation-prod/00-namespace.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: case-notes-to-probation-prod
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
```

Comment:
Should "is-production" be "true" here?

Reply:
:man_facepalming: hadn't spotted that, now fixed (and two other of envs that didn't have the flag set either)